PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/RootLayoutController.java

```diff
@@ -44,31 +62,99 @@
   private SplitPane mainPane;
   private final Pane graphPane = new Pane();
 
-  private static RootLayoutController controller;
   private TreeManager treeManager;
   private HeatmapManager heatmapManager;
   private SelectionManager selectionManager;
   private boolean zoomOutButtonDisabled = true;
+  private OriginalGraph graph;
+
+  private GraphOrdererThread mainGraphOrder;
+  private final DrawComparedGraphs drawGraphs = DrawComparedGraphs.loadView();
 
   /**
    * Initializes the controller class.
+   *
+   * @param location  unused variable.
+   * @param resources unused variable.
    */
-  public void initialize() {
-    assert (controller == null);
-    controller = this;
+  @Override
+  public void initialize(URL location, ResourceBundle resources) {
     heatmapManager = new HeatmapManager(heatmapPane);
     initializeSelectionManager();
     initializeTreeIcon();
     initializeGraphIcon();
-    new DrawGraph().drawGraph(graphPane);
+    graph = new GfaReader("TB328.gfa").read();
```

Comment:
Correct, see #100. This PR has lots of conflicts I'd like resolve with 4 eyes as both changesets are pretty big and move code around.